### PR TITLE
Redesign game menu

### PR DESCRIPTION
**Pull request by Heartfyre**
gamemenu

### CODE REVIEW

**Comment by Ajackster** [comment]
Looks great, just a couple items. Talked to @codecorsair and @jamkoo 
1.) Can we have an X box instead of the "Close Menu" button?
2.) The title and button fonts are off, the Settings menu has the correct fonts so just copy those
3.) Can we make the widget a bit darker, similar to the darkness of the Settings menu? Attached is a quick mockup of the darkness

kooscreen1529954557 png

**Comment by Heartfyre** [comment]
menu

Added a darker background, reworked the title graphics, so it fits better with the smaller window, reworked the buttons and the glow (they're bigger than the main ones), changed font-size and color, so it fits the other windows (settings). I also removed the close button from the footer, and the footer as well , and added the close button the the top right corner.

File: game/hud/src/components/GameMenu/index.tsx [hunk 1]
```diff
@@ -17,59 +17,120 @@ export interface GameMenuStyle {
...
-const Footer = styled('div')`
-  display: flex;
-  justify-content: center;
-  padding: 5px 0;
-  background-color: #202020;
-  color: white;
-  border-top: 1px solid ${utils.lightenColor('#202020', 30)}
+const MenuCorner = styled('div')` {
+  position: absolute;
+  min-width: 320px;
```
Comment (Ajackster) [inline game/hud/src/components/GameMenu/index.tsx]: Any reason you set absolute width and heights for these guys? Could just use 100% for the dimensions

Comment (Ajackster) [reply game/hud/src/components/GameMenu/index.tsx]: You don't need to update this, I can do on local comp if there's no problems

**Comment by Ajackster** [comment]
I'm gnna merge this in and then make some small modifications. The modifications are
1.) A little less padding on the sides
2.) Reuse background image that MOTD is using

Thanks @Heartfyre !!

**Pull request merged**